src/a11y-dialog.js

```diff
@@ -1,283 +1,237 @@
 import focusableSelectors from 'focusable-selectors'
 
-var TAB_KEY = 'Tab'
-var ESCAPE_KEY = 'Escape'
+const TAB_KEY = 'Tab'
+const ESCAPE_KEY = 'Escape'
```

Comment:
Also wondering whether we should just call these `TAB` and `ESCAPE`. This can be done separately though.